### DISCUSSION TITLE
Dark mode missing?

### DISCUSSION THREAD

**Discussion by penge1510** [nocodb/nocodb#4647] [Q&A] [answered]
Hi All, 

Today I've just upgraded NocoDB from 0.92.4 to the latest version. 
First thing that I can't find the dark theme mode. Is this removed? 
I couldn't find related answers or is there something that I miss why can't find this feature.

Thanks in advance!

**Comment by o1lab** [comment, marked as answer]
Its not available at the moment.

**Reply by o1lab** [reply]
Cuz this is rewrite of frontend using a new framework. But will think about it in future.

**Reply by penge1510** [reply]
Okay, thanks!
